### ISSUE TITLE
Theme User Preferences: "additional property error"

### ISSUE THREAD

**Issue by mattzque** [jupyterlab/jupyterlab#9430]
I'm trying to modify the theme via the Jupyterlab UI. When I try to add an "override" of "content-font-family" to change the font family it doesn't let me save it because of some validation error (additional property error).

The UI is found under Settings -> Advanced Settings Editor -> Theme -> User Preferences.

It might be this line here:

https://github.com/jupyterlab/jupyterlab/blob/9d8650326bf47805e0aa3a51fe8e3cabf1f5d346/packages/apputils-extension/schema/themes.json#L10

Should this not allow other properties as well, such as the content-font-family property?

**Comment by telamonian** [comment]
Nope, this is intentional.  The only currently supported theme overrides are the ones explicitly listed in the default theme user settings:

![image](https://user-images.githubusercontent.com/2263641/101271116-16893680-374e-11eb-90fe-4fe8ee25b1cd.png)

We decided that it would be a Bad Idea to surface the entire confusing (and somewhat poorly organized) sea of CSS vars that currently make up a standard [jlab theme](https://github.com/jupyterlab/jupyterlab/blob/master/packages/theme-light-extension/style/variables.css) to average users. So for the overrides we went with a limited and incremental approach.

That being said, I wouldn't say no to a PR that adds some new font-family overrides to the existing set. You'd just have to add appropriate definitions to that `theme.json` schema you linked to.
